src/index.js

```diff
@@ -1,2 +1,94 @@
 // import _ from 'lodash';
 import './style.css';
+
+const baseURL = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api';
+let gameId;
+
+const createGame = async () => {
+  try {
+    const response = await fetch(`${baseURL}/games/`, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify({ name: 'My cool new game' }),
+    });
+    const game = await response.json();
+    // eslint-disable-next-line prefer-destructuring
+    gameId = game.result.split(' ')[3];
+    console.log('Game ID:', gameId);
+  } catch (error) {
+    console.error('Error creating game:', error);
+  }
+};
+
+createGame();
+
+const refreshScores = async () => {
+  try {
+    const response = await fetch(`${baseURL}/games/${gameId}/scores`);
+    const scores = await response.json();
+    console.log('Scores:', scores);
+  } catch (error) {
+    console.error('Error fetching scores:', error);
+  }
+};
```

Comment:
It is a requirement that when the refresh button is clicked, all scores that have been sent or saved to the API endpoint are retrieved and rendered in the scores list😉. But currently, when we click the refresh button, nothing happens which means it doesn't call the GET API endpoint😉. kindly do well to fix this issue👍🏼.

<img width="971" alt="image" src="https://github.com/hydaline-code/Leaderboard/assets/64643995/96ff3adf-0edc-4b38-a676-635f073f29c6">